addons/delete-others/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Delete other assets",
```

Comment:
We could also rename it to `Delete other costumes`, as that's what most users will use it for.

Reply:
It can still be used for costumes though, so it wouldn't really make sense.

Reply:
"Delete other costumes or sounds" would be better for searchability. The addon appears when searching for "delete other costumes" or "delete other sounds" on master, but not in this PR because it removes the word "other" from the description.